scripts/custom_targeting_to_coverage.R

```diff
@@ -0,0 +1,74 @@
+##this script creates a coverage table (proportion of target population vaccinated at admin 0 level) based on information inside 
+##a custom-made targeting table (number of vaccinated people at admin 2 level)
+
+## Load packages using Kaiyue's approach for other scripts in the repo
+chooseCRANmirror(ind = 77) #specify the mirror so that the packages can be successfully installed in the non-interactive way
+
+package_list <- c(
+  "GADMTools", 
+  "rgdal", 
+  "drat", 
+  "roxygen2", 
+  "data.table",
+  "dplyr",
+  "exactextractr",
+  "fasterize",
+  "optparse",
+  "purrr",
+  "raster",
+  "readr",
+  "sf",
+  "stringr",
+  "readxl",
+  "tibble",
+  "tidyr",
+  "yaml",
+  "ISOcodes"
+)
+
+for (package in package_list) {
+  if (!require(package = package, character.only = T)) {
+    install.packages(pkgs = package)
+    library(package = package, character.only = T)
+  }
+  detach(pos = which(grepl(package, search())))
+}
+
+##load custom targeting table to calculate proportion vaccinated with one and two doses and get admin 0 population
+
+targeting_table <- read_csv("input_data/custom_targeting.csv")  %>%
```

Comment:
I think we should leave a comment that the custom targeting table needs to be specified by the user prior to running the script (rather than hardcoding a file name and path)

We should also provide a template for custom_targeting.csv -- what are the column names and types that need to be included for the file to run correctly?